docker/service/src/main/java/uk/nhs/adaptors/scr/clients/SandboxSpineClient.java

```diff
@@ -2,16 +2,22 @@
 
 import lombok.SneakyThrows;
 import org.apache.commons.io.IOUtils;
+import org.apache.http.Header;
+import org.apache.http.message.BasicHeader;
 import org.springframework.core.io.ClassPathResource;
 import uk.nhs.adaptors.scr.clients.SpineHttpClient.Response;
 import uk.nhs.adaptors.scr.models.ProcessingResult;
 
 import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.springframework.http.HttpStatus.ACCEPTED;
 import static org.springframework.http.HttpStatus.OK;
+import static uk.nhs.adaptors.scr.clients.SpineHttpClient.CONTENT_LOCATION_HEADER;
+import static uk.nhs.adaptors.scr.clients.SpineHttpClient.RETRY_AFTER_HEADER;
 
 public class SandboxSpineClient implements SpineClientContract {
 
     private static final String EVENT_LIST_QUERY_RESPONSE = "mock-spine/QUPC_IN200000SM04/success.xml";
+    private static final String POST_SCR_PROCESSED_RESPONSE = "mock-spine/MCCI_IN010000UK13/success.xml";
```

Comment:
Can we rename it? Maybe UPLOAD_SCR_POLLING_RESPONSE?

Reply:
Renamed.